src/main/java/edu/greenblitz/utils/GBMath.java

```diff
@@ -1,20 +1,108 @@
 package edu.greenblitz.utils;
 
+import edu.wpi.first.math.Pair;
+import edu.wpi.first.math.geometry.Translation2d;
+
 public class GBMath {
-	/**
-	 *
-	 * modulo (2,6) -> 0
-	 * modulo (2,3) -> 1
-	 * modulo (-2,3) -> 1
-	 * modulo (3,-2) -> -1 -> 1
-	 *
-	 * @param x the number to be modulu-ed
-	 * @param y the modulo base
-	 * @return the correct modulo result
-	 */
-	public static double absoluteModulo(double x, double y) {
-		return ((x % y) + y) % y;
-	}
-	
-	
+
+    public static double limit(double velocity, double maxMagnitude) {
```

Comment:
Since this function is in a utils class, it should have more generic names for the general purpose of this function.
Also, you don't have to change this part, but this function is usually called 'clamp'

Reply:
but the function return the value that it gets after limitation so what can be more generic name?